src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/ExceptionRecordMapper.java

```diff
@@ -46,10 +51,22 @@ public ExceptionRecord mapEnvelope(Envelope envelope) {
             envelope.id,
             CollectionUtils.isEmpty(envelope.payments) ? NO : YES,
             CollectionUtils.isEmpty(envelope.payments) ? NO : YES,
-            envelope.caseRef
+            isBlank(envelope.caseRef) ? null : envelope.caseRef,
+            isBlank(envelope.legacyCaseRef) ? null : envelope.legacyCaseRef,
+            setDisplayCaseReferenceFlag(envelope.caseRef, envelope.classification),
+            setDisplayCaseReferenceFlag(envelope.legacyCaseRef, envelope.classification)
         );
     }
 
+    private String setDisplayCaseReferenceFlag(String caseRef, Classification classification) {
+        if (isNotBlank(caseRef) && (SUPPLEMENTARY_EVIDENCE.equals(classification)
+            || SUPPLEMENTARY_EVIDENCE_WITH_OCR.equals(classification))
```

Comment:
We could try and extract long classification comparison to separate variable. Let's say

```java
EnumSet<Classification> allowedClassifications = EnumSet.of(
    SUPPLEMENTARY_EVIDENCE,
    SUPPLEMENTARY_EVIDENCE_WITH_OCR
);

if (isNotBlank(caseRef) && allowedClassifications.contains(classification)) { ... }
```